src/mono/wasm/debugger/DebuggerTestSuite/MonoJsTests.cs

```diff
@@ -136,7 +136,7 @@ async Task AssemblyLoadedEventTest(string asm_name, string asm_path, string pdb_
                             :  await Task.FromResult(ProtocolEventHandlerReturn.KeepHandler);
             });
 
-            byte[] bytes = File.ReadAllBytes(asm_path);
+            byte[] bytes = File.Exists(asm_path) ? File.ReadAllBytes(asm_path) : File.ReadAllBytes(Path.ChangeExtension(asm_path, ".webcil")); // hack!
```

Comment:
Maybe we need to take look at this code?
https://github.com/dotnet/aspnetcore/blob/main/src/Components/WebAssembly/WebAssembly/src/Services/LazyAssemblyLoader.cs
Not sure if the Blazor Lazy Assembly Loader will continue working with the dll's using .webcil extension.

Reply:
I thought the workload build tests would try it.  but I can try manually, too.

Reply:
Hmm... ah ok, lazy loading is pretty complicated actually. https://learn.microsoft.com/en-us/aspnet/core/blazor/webassembly-lazy-load-assemblies?view=aspnetcore-7.0
I'll try the example from the documentation. you're probably right it needs some  changes in the blazor service.

Also, we might not need this working in the first preview.